packages/example/playwright/1.without-metamask.test.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai'
```

Comment:
are `1.*` and `2.*` name important for order? can you please add a comment if so